Savefile andi v1

schallakuchen opened the pull request:
Create Methods to Save and Load Sound object list

schallakuchen committed:
Sound Dictionary + subfolder support

b3llatrixs committed:
test-upload to merge files with WinSCP

b3llatrixs committed:
deleted test_document.txt

b3llatrixs committed:
updated files to the latest version  and deleted obsolete files

b3llatrixs committed:
fixed play-command,added categories-command, tried to make !list as tabular-format

schallakuchen committed:
import und export sounds erstellt

Create simple import and export methods to save and load a serialized object_sound_list to and from a file using pickle

samuscherer commented:
Hi! Thank you for your contribution. I'll check out your changes as soon as possible and let you know if there's anything that needs to be changed 😊

schallakuchen commented:
Hello,

It is possible, that I branched wrongly 😄.

Just for your information. I was getting started in python programming and
a friend of mine and me started to tinker around with the bot. We are new
to GIT and will need a bit of time to get into the workflow.

Currently my last push to the sidebranch has many Bugs and is incomplete
right now.
However, it is going to be great at the end - eventually.


Best regards,
Andreas

Samuel Scherer ***@***.***> schrieb am Mo., 12. Apr. 2021,
08:13:

> Hi! Thank you for your contribution. I'll check out your changes as soon
> as possible and let you know if there's anything that needs to be changed
> 😊
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/samuscherer/discord-soundboard/pull/14#issuecomment-817515154>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJK7WF47ZOIYTP4VQ2BGC2DTIKFPXANCNFSM42YAHRFA>
> .
>

schallakuchen committed:
Add save_file history

save file history and error handling
add new entries to config.json

schallakuchen committed:
Init function and first startup

Improved stability for first bot startup on a new system
Add Init function to keep track of initialization process.

schallakuchen commented:
Pull request will be closed since it was done by accident.
